interfaces/smtp/v0/README.md

```diff
@@ -0,0 +1,64 @@
+# `smtp`
+
+## Overview
+
+This relation interface describes the expected behavior between of any charm claiming to be able to interface with a SMTP Provider and the SMTP Provider itself.
```

Comment:
Aside from not being grammatically correct, this whole sentence is confusing.

In ingress we have: `This relation interface describes the expected behavior of any charm claiming to be able to provide or consume ingress data.`

We didn't yet quite agree on a generic formulation for what this should look like, and also, it's kind of repeated information in every README so IMHO we could also decide to drop this section altogether @simskij @gruyaume @benhoyt @tmihoc ?

For now, we could rephrase it to something more correct/informative:
```suggestion
This relation interface specification describes the expected behavior of charms integrating over the `smtp` Juju interface to provide or consume SMTP data.
```

Reply:
see #127 for further discussion, dropping this for now